coderdojochi/settings.py

```diff
@@ -368,7 +374,3 @@ def custom_show_toolbar(request):
         'TAG': 'div',
         'ENABLE_STACKTRACES': True,
     }
-
-
-# Activate Django-Heroku.
-django_heroku.settings(locals(), staticfiles=False)
```

Comment:
Doesn't this break the staging builds on heroku since the staging site has `DEBUG = True`?

Reply:
Probably! I forgot we have DEBUG on in staging.